### PR TITLE
Update README for dev instructions

### PR DESCRIPTION
**Pull request by jessicard**
Add some docs and dev instructions to README. As a note, bots are kinda confusing to get running locally! 😵‍💫 As @sampoder said, the easiest way to update Scrappy is by pushing to `main` and letting it deploy.

Fixes https://github.com/hackclub/scrappy/issues/311.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,54 @@
-# scrappy
+# Scrappy the Slack Bot
 
 ![build](https://github.com/hackclub/scrappy/workflows/build/badge.svg)
 
-the slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). to get started developing, ask @sampoder to be added to the `scrappy (dev)` app on Slack and for a `.env` file. then use `ngrok` to forward your local instance onto the world wide web.
+Scrappy is the Slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). Scrappy automatically generates your Scrapbook and Scrapbook posts via Slack messages. For more information about how to sign up,
```
Comment (sampoder) [inline README.md:5]: `For more information about how to sign up,` <- this sentence probably needs finishing

File: README.md [hunk 2]
```diff
@@ -1,9 +1,54 @@
-# scrappy
+# Scrappy the Slack Bot
```
Comment (sampoder) [inline README.md:1]:
```suggestion
# Scrappy
```
I'm leaning towards keeping it at Scrappy but idk tbh

File: README.md (hunk 1 above)
Comment (sampoder) [inline README.md:5]:
```suggestion
Scrappy is the Slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). Scrappy generates your Scrapbook and Scrapbook posts via Slack messages. For more information about how to sign up,
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,54 @@
...
+- `/scrappy-togglestreaks all`: opts out of streaks completely
+- `/scrappy-open`: opens your scrapbook (or another user's if you specify a username)
+- `/scrappy-setcss`: adds a custom CSS file to your scrapbook profile. Check out this cool example!
+- `/scrappy-setdomain`: links a custom domain to your scrapbook profile, e.g. [https://zachlatta.com](https://zachlatta.com)
+- `/scrappy-setusername`: change your profile username
+- `/scrappy-setaudio`: links an audio file to your Scrapbook. [See an example here](https://scrapbook.hackclub.com/matthew)!
+- `/scrappy-setwebhook`: create a Scrappy Webhook we will make a blank fetch request to this URL every time you post
+- `/scrappy-webring`: adds or removes someone to your webring
+- *Remove* a post: delete the Slack message and Scrappy will automatically update for you
+- *Edit* a post: edit the Slack message and it will automatically update for you
```
Comment (sampoder) [inline README.md:22]: Might be worth adding that reacting with `:scrappy:` can add a post

File: README.md [hunk 4]
```diff
@@ -1,9 +1,54 @@
...
+
+## Contributing
+
+Contributions are encouraged and welcome! There are two GitHub repositories that contain code for Scrapbook: the [Scrapbook website](https://github.com/hackclub/scrapbook#contributing) and [Scrappy the Slack bot](https://github.com/hackclub/scrappy#contributing). Each repository has a section on contributing guidelines and how to run each project locally.
```
Comment (sampoder) [inline README.md:26]:
```suggestion
Contributions are encouraged and welcome! There are two GitHub repositories that contain code for Scrapbook: the [Scrapbook website](https://github.com/hackclub/scrapbook#contributing) and [Scrappy (the Slack bot)](https://github.com/hackclub/scrappy#contributing). Each repository has a section on contributing guidelines and how to run each project locally.
```

File: README.md (hunk 2 above)
Comment (jessicard) [reply README.md:1]: Yeah that works for me!

File: README.md (hunk 1 above)
Comment (jessicard) [reply README.md:5]: Very nice 👏

File: README.md (hunk 4 above)
Comment (jessicard) [reply README.md]: Lgtm!

File: README.md (hunk 1 above)
Comment (jessicard) [reply README.md]: LOL oops [f709dbd](https://github.com/hackclub/scrappy/pull/317/commits/f709dbda96bc28ffe42a84d9500f169fddf8db27)

File: README.md (hunk 3 above)
Comment (jessicard) [reply README.md:22]: Such a good feature [94669b5](https://github.com/hackclub/scrappy/pull/317/commits/94669b5e46b6f191f45b4483aff07cbf3070b4f8)

**Comment by jessicard** [comment]
@sampoder Ready for another review! 👀

File: README.md [hunk 5]
```diff
@@ -1,9 +1,55 @@
-# scrappy
+# Scrappy
 
 ![build](https://github.com/hackclub/scrappy/workflows/build/badge.svg)
 
-the slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). to get started developing, ask @sampoder to be added to the `scrappy (dev)` app on Slack and for a `.env` file. then use `ngrok` to forward your local instance onto the world wide web.
+Scrappy is the Slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). Scrappy generates your Scrapbook and Scrapbook posts via Slack messages. For more information about how to sign up for Scrapbook, check out the [about page]([scrapbook.hackclub.com](https://scrapbook.hackclub.com).
```
Comment (sampoder) [inline README.md:5]:
```suggestion
Scrappy is the Slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). Scrappy generates your Scrapbook and Scrapbook posts via Slack messages. For more information about how to sign up for Scrapbook, check out the [about page](https://scrapbook.hackclub.com/about).
```

File: README.md [hunk 6]
```diff
@@ -1,9 +1,55 @@
...
+- `/scrappy-open`: opens your scrapbook (or another user's if you specify a username)
+- `/scrappy-setcss`: adds a custom CSS file to your scrapbook profile. Check out this cool example!
+- `/scrappy-setdomain`: links a custom domain to your scrapbook profile, e.g. [https://zachlatta.com](https://zachlatta.com)
+- `/scrappy-setusername`: change your profile username
+- `/scrappy-setaudio`: links an audio file to your Scrapbook. [See an example here](https://scrapbook.hackclub.com/matthew)!
+- `/scrappy-setwebhook`: create a Scrappy Webhook we will make a blank fetch request to this URL every time you post
+- `/scrappy-webring`: adds or removes someone to your webring
+- *Remove* a post: delete the Slack message and Scrappy will automatically update for you
+- *Edit* a post: edit the Slack message and it will automatically update for you
+- *Post* an existing Slack message to Scrapbook by reacting to it with the `:scrappy:` emoji (Note: If it isn't working, make sure Scrappy is added to the channel by mentioning `@scrappy`)
```
Comment (sampoder) [inline README.md:23]:
```suggestion
- *Post* a message to the `#scrapbook` channel or add an existing Slack message to Scrapbook by reacting to it with the `:scrappy:` emoji (Note: If it isn't working, make sure Scrappy is added to the channel by mentioning `@scrappy`)
```

File: README.md (hunk 5 above)
Comment (jessicard) [reply README.md:5]: 🤦 Thank you

File: README.md (hunk 6 above)
Comment (jessicard) [reply README.md:23]: Oooh that's better 👍

**Comment by jessicard** [comment]
@sampoder Thanks for all the review! 🙏 Any other changes you think? If not, feel free to merge away!

**Review by sampoder — APPROVED** [review]
let's get this merged!

**Pull request merged**